servers/tests/org.fusesource.ide.server.tests/src/main/java/org/fusesource/ide/server/tests/util/FuseESBMockRuntimeCreationUtil.java

```diff
@@ -24,10 +30,19 @@
 	
 	public static final String FUSEESB_60 = "org.fusesource.ide.fuseesb.runtime.60";
 	public static final String FUSEESB_61 = "org.fusesource.ide.fuseesb.runtime.61";
-	public static final String FUSEESB_6x = "org.fusesource.ide.fuseesb.runtime.6x";
+	public static final String FUSEESB_62 = "org.fusesource.ide.fuseesb.runtime.62";
+	public static final String FUSEESB_63 = "org.fusesource.ide.fuseesb.runtime.63";
+	public static final String FUSEESB_6X = "org.fusesource.ide.fuseesb.runtime.6x";
+	
+	public static final String FUSEESB_70 = "org.fusesource.ide.fuseesb.runtime.70";
+	public static final String FUSEESB_7X = "org.fusesource.ide.fuseesb.runtime.7x";
 	
 	public static final String[] SUPPORTED_6X_RUNTIMES = new String[] {
-		FUSEESB_60, FUSEESB_61
+		FUSEESB_60, FUSEESB_61, FUSEESB_62, FUSEESB_63
```

Comment:
it is often more convenient to use List<String>
	public static final List<String> SUPPORTED_6X_RUNTIMES = Arrays.asList(FUSEESB_60, FUSEESB_61, FUSEESB_62, FUSEESB_63);

Reply:
up